gnocchi/opts.py

```diff
@@ -193,6 +193,9 @@ def list_opts():
                        default=10, min=0,
                        help='Number of seconds before timeout when attempting '
                             'to do some operations.'),
+            cfg.StrOpt('uwsgi-path',
```

Comment:
all other options in this group uses an underscore here, so perhaps ``uwsgi_path`` instead for consistency

Reply:
Done